### PR TITLE
Question Page Styling

### PR DESCRIPTION
**Pull request by alexisljn**
| Q             | A
| ------------- | ---
| Is bugfix?    | No
| New feature?  | No
| Fixed issues  | Fixes # if any

Styling for the question Page

### CODE REVIEW

**Comment by alexisljn** [comment]
Review Required

File: frontend/src/css/style.css [hunk 1]
```diff
@@ -562,6 +562,70 @@ button:disabled {
...
+.question-header-info-box {
+    border-radius: 10px;
+    text-align: center;
+    padding: 5px 65px;
+}
+
+.question-header-info-box:nth-child(2)
+{
+    margin-top: 10px;
+}
```
Comment (aekix) [inline frontend/src/css/style.css:587]: Maybe add a text-transform: capitalize; To ensure the uppercase of the first letter of the question type?

Comment (alexisljn) [reply frontend/src/css/style.css:587]: Good idea

**Review by aekix — APPROVED** [review]
nice

**Pull request merged**